generate-xml/domain.nix

```diff
@@ -326,6 +326,7 @@ let
                     (subelem "readonly" [ ] [ ])
                     addresselem
                     (subelem "boot" [ (subattr "order" typeInt) ] [ ])
+                    (subelem "serial" [ ] (sub "value" typeString))
```

Comment:
I think this should be:
`(subelem "serial" [ ] typeString)`

Reply:
I wanted to do that but I couldn't figure it out, thanks!